### PR TITLE
[MXNet][Sanity Test]Updated packages for MXNet

### PR DESCRIPTION
**Pull request by TusharKanekiDey**
*Issue #, if available:*
MXNet build is failing for PY2 due to setuptools packages
Sanity test is failing due to various outdated packages

## Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [build] | [test] | [build, test] | [ec2, ecs, eks, sagemaker]
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

*Description:*

*Tests run:*

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: mxnet/training/docker/1.6.0/py3/Dockerfile.gpu [hunk 1]
```diff
@@ -151,16 +151,18 @@ RUN ${PIP} install --no-cache --upgrade \
...
-    git+git://github.com/dmlc/gluon-nlp.git@v0.9.0 \
-    urllib3==1.24 \
+    gluonnlp==0.9.1 \
+	gluoncv==0.6.0 \
+	urllib3==1.25.8 \
     python-dateutil==2.8.0 \
     tqdm==4.39.0 \
-    sagemaker-experiments==0.1.3 \
-    PyYAML==5.1.2 \
+	sagemaker-experiments==0.1.7 \
```
Comment (TusharKanekiDey) [inline mxnet/training/docker/1.6.0/py3/Dockerfile.gpu:159]: Sorry for syntax error, will correct it before merging it.

File: mxnet/inference/docker/1.6.0/py3/Dockerfile.gpu [hunk 2]
```diff
@@ -67,11 +67,12 @@ WORKDIR /
 RUN ${PIP} install --no-cache-dir \
     ${MX_URL} \
     git+git://github.com/dmlc/gluon-nlp.git@v0.9.0 \
+    gluoncv==0.6.0 \
     mxnet-model-server==$MMS_VERSION \
     keras-mxnet==2.2.4.1 \
     numpy==1.17.4 \
     onnx==1.4.1 \
-    sagemaker-mxnet-inference
+    "sagemaker-mxnet-inference<2"
```
Comment (YYStreet) [inline mxnet/inference/docker/1.6.0/py3/Dockerfile.gpu:75]: Do not have much context on it. Why this "<2" is required? I find it happens to several Dockerfiles

**Comment by YYStreet** [comment]
Please resolve the indentation issue before merging. Also get another approval first.

**Comment by arjkesh** [comment]
Is this just syncing with the public mxnet Dockerfiles?

**Comment by TusharKanekiDey** [comment]
@arjkesh Yes, It is just sync with public dockerfile and pinning setuptool version to avoid build failure.

File: mxnet/training/docker/1.6.0/py3/Dockerfile.gpu (hunk 1 above)
Comment (nskool) [reply mxnet/training/docker/1.6.0/py3/Dockerfile.gpu:159]: You'll have to fix any changes before the final approvals, else the approval is dismissed.

File: mxnet/inference/docker/1.6.0/py3/Dockerfile.gpu (hunk 2 above)
Comment (TusharKanekiDey) [reply mxnet/inference/docker/1.6.0/py3/Dockerfile.gpu:75]: Synced up about this in offline.

**Pull request merged**